plutus-contract/src/Wallet/Emulator/Folds.hs

```diff
@@ -120,6 +121,10 @@ scriptEvents = preMapMaybe (preview (eteEvent . chainEvent) >=> getEvent) (conca
 walletTxBalanceEvents :: EmulatorEventFold [UnbalancedTx]
 walletTxBalanceEvents = preMapMaybe (preview (eteEvent . walletEvent' . _2 . _TxBalanceLog . _BalancingUnbalancedTx)) L.list
 
+-- | Min lovelace of 'txOut's from adjusted unbalanced transactions
+walletAdjustedTxEvents :: EmulatorEventFold [(Wallet, [Value])]
```

Comment:
I'm guessing this includes ALL transaction outputs of transaction's for which `adjustUnbalancedTx` was called on? Even the outputs which didn't need adjusting are listed?

Reply:
Yep, that's correct. At the beginning I even tried to use it — to understand what transactions were adjusted to calculate wallets funds. But turned out it's too hard and requires more info.